core/runtime/ops.rs

```diff
@@ -343,6 +343,33 @@ pub fn serde_v8_to_rust<'a, T: Deserialize<'a>>(
   from_v8(scope, input)
 }
 
+/// Retrieve a [`serde_v8::V8Slice`] from a value.
+#[allow(clippy::result_unit_err)]
+pub fn to_nonresizable_v8_slice(
+  scope: &mut v8::HandleScope,
+  input: v8::Local<v8::Value>,
+) -> Result<serde_v8::V8Slice, ()> {
+  let (buf, offset, length) =
+    if let Ok(buf) = v8::Local::<v8::ArrayBufferView>::try_from(input) {
+      let Some(buffer) = buf.buffer(scope) else {
+      return Err(());
+    };
```

Comment:
This formatting seems funky

Reply:
Huh.. looks like the rustfmt bug with else statements. Will fix.